README.rst

```diff
@@ -36,14 +36,24 @@ The following versions of Python are supported:
 
 Basic Usage
 -------------
+You can expect a ``Client`` instance to have reasonable defaults, like the Fauna endpoint ``db.fauna.com`` and a global HTTP client, but you will always need to supply a secret.
```

Comment:
this reads as if arg `secret=""` is required

Reply:
The next line says how. What do you propose?

Reply:
how about 

```suggestion
You can expect a ``Client`` instance to have reasonable defaults, like the Fauna endpoint ``db.fauna.com`` and a global HTTP client, but you will always need to configure a secret.
```